packages/rx_bloc_cli/mason_templates/bricks/feature_login/__brick__/lib/feature_login/views/login_page.dart

```diff
@@ -28,24 +32,38 @@ class LoginPage extends StatelessWidget {
               horizontal: context.designSystem.spacing.xxl2,
             ),
             child: {{^enable_login}}{{^enable_social_logins}}const {{/enable_social_logins}}{{/enable_login}}Column(
-              mainAxisAlignment: MainAxisAlignment.center,
+              mainAxisAlignment: MainAxisAlignment.spaceEvenly,
               crossAxisAlignment: CrossAxisAlignment.stretch,
-              children: [{{#enable_login}}
-                LoginForm(
-                  title: context.l10n.featureLogin.loginCredentialsHint,
-                ),{{/enable_login}}{{#enable_social_logins}}
-                SizedBox(height: context.designSystem.spacing.xs),
-                const FacebookLoginWidget(),
-                if (Platform.isIOS)
-                  Column(
-                    children: [
-                      SizedBox(height: context.designSystem.spacing.xs),
-                      const AppleLoginWidget(),
-                    ],
+              children: [
```

Comment:
Please confirm the latest changes from @nikolay-vasilev-prime are incorporated in this branch

Reply:
Yes, it took some time to merge all changes together, and it would be safer if somebody else double checks this, but it should unify both implementations.

Reply:
@nikolay-vasilev-prime, can you please assist here ?

Reply:
I've replaced the previously added Register button with the new design, just added the handling to make the new Register button work